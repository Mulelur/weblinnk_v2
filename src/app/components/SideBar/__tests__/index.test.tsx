import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';
import { SideBar } from '../index';

const shallowRenderer = createRenderer();

describe('<SideBar />', () => {
  it('should match snapshot', () => {
    shallowRenderer.render(<SideBar />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
