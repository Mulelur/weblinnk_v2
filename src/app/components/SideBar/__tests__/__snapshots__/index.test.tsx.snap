// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SideBar /> should match snapshot 1`] = `
<SideBar__Wrapper>
  <Logo />
  <SideBar__Divider />
  <Nav />
  <Upgrade />
</SideBar__Wrapper>
`;
